src/main/resources/data-repository-openapi.yaml

```diff
@@ -276,6 +288,24 @@ paths:
           description: Not found - job id does not exist
 
 
+  '/api/repository/v1/jobs/{id}/result':
+    get:  ## retrieve
+      description: Retrieve a job's result by id
+      operationId: retrieveJobResult
+      tags:
+      - repository
+      parameters:
+      - $ref: '#/parameters/Id'
+      responses:
+        303:
```

Comment:
I don't think we can specify the response here. The result is returning the result from the async call, not the result of any jobs thing. For example, if we are doing create study, than the response would be a 201 code / StudySummaryModel on success and some 4xx/5xx code / ErrorModel on failure.